spring-xd-module/src/main/java/org/springframework/xd/module/core/ModuleFactory.java

```diff
@@ -135,6 +144,18 @@ else if (JavaConfiguredModule.basePackages(moduleDefinition, moduleClassLoader).
 		return null;
 	}
 
+	private Module createSparkModule(ModuleDescriptor moduleDescriptor, ModuleOptions moduleOptions,
```

Comment:
OK, this seems to confirm what I was thinking: I believe this should move to a case in SimpleModuleCreator, not here